mod/wet4/views/default/forms/polls/vote.php

```diff
@@ -50,7 +50,7 @@
 	$responses4 = gc_explode_translation($value, $lang);
 
 
-$form_body .="  <input type='radio'  class = 'mrgn-rght-sm' id='responses-".$responses4."' name='response3' value='".$value."'> <label for='responses-".$responses4."'>".$responses4."</label><br>";
+$form_body .="  <input type='radio'  class = 'mrgn-rght-sm' id='responses-".$responses4."' name='response3' value=".$value."> <label for='responses-".$responses4."'>".$responses4."</label><br>";
```

Comment:
`value='". htmlEntities($value, ENT_QUOTES) ."'` seems to be the safe way to do this

Reply:
The polls mod for whatever reason does string matching to figure out what option a vote is for... so the value would need to be identical here to the json string that's used to store the text for the vote response/option. I checked some of the more recent versions and they all seem to stick to this way of storing votes.

Reply:
Wouldn't any value with a space blow up then? Or is it better to filter out poll options with any single quotes instead.

Reply:
Did a bit of testing and it looks like elgg does actually parse the html entities in the value before storing it so your suggestion would the the way to go, thanks!